docs/docusaurus/src/css/overview_pages.scss

```diff
@@ -38,14 +38,24 @@
   max-width: 40rem;
 }
 
-.generated-index-page header {
-  @include overview-card-container;
+.generated-index-page {
+  .theme-doc-breadcrumbs {
+    display: none;
+  }
 
-  h1 {
-    @include overview-card-title;
+  & > div {
+    max-width: unset !important;
```

Comment:
Note: unfortunately Docusaurus sets a `max-width` of 75% percent in this div and the original rule has an `!important`, so I had to include an `!important` in my rule as well :cry: